CHANGELOG.rst

```diff
@@ -30,6 +30,43 @@ History
 
 * **Breaking Change**:  Change signature of hook ``update_internal_deposition_layer``. Now, instead of the deposition thickness, it is returned the phase index of phase being deposited and the thickness variation rate.
 * **Breaking Change**:  Change signature of hook ``calculate_relative_emulsion_viscosity`` and function ``get_relative_emulsion_viscosity``. Indices of continuous and dispersed fields were removed.
+* **Breaking Change**: Change in ``PhysicsDescription``: split emulsion model into relative viscosity, droplet size, and inversion point models and a flag to activate the emulsion models.
```

Comment:
You probably want to rebase on master because those changes will enter on `0.14.0` (`0.13.0` was already released).
And from my experience the merges "version bump on change log" and changes "on master before release" do not cause conflicts (as in need to be hand updated before merge) but the changes being listed on an already released version.